.features/pending/convert.md

```diff
@@ -0,0 +1,9 @@
+Description: `convert` CLI command to convert to new workflow format
+Authors: [Alan Clucas](https://github.com/Joibel)
+Component: CLI
+Issues: 14977
+
+A new CLI command `convert` which will convert Workflows, CronWorkflows, and (Cluster)WorkflowTemplates to the new format.
```

Comment:
I would suggest `migrate` as the CLI command name. `convert` sounds like it might convert between `Workflows`/`CronWorkflows`/etc. (Might even be a good feature for later.)

Reply:
In the PR description I mention the name comes from `kubectl convert` which 
> Convert config files between different API versions. Both YAML and JSON formats are accepted. Note - requires kubectl-convert plugin to be installed.